modules/bitgo/example/ts/btc/make-lightning-deposit.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Create lightning deposit address and deposit on-chain funds to a wallet's lightning balance.
+ *
+ * Copyright 2022 BitGo, Inc.  All Rights Reserved.
+ */
+const BitGoJS = require('bitgo');
+
+// change this to env: 'production' when you are ready for production
+const env = 'test';
+// change coin to 'btc' when working with production
+const coin = 'tbtc';
+
+const bitgo = new BitGoJS.BitGo({ env });
+
+// set your access token here
+const accessToken = '';
+
+// set your wallet from the YYYYY parameter here in the URL on app.bitgo-test.com
+// https://test.bitgo.com/enterprise/XXXXXXXXX/coin/tbtc/YYYYY/transactions
```

Comment:
// https://api.bitgo.com/enterprise/XXXXXXXXX/coin/tbtc/YYYYY/transactions
Where is the documentation for this endpoint?